climada_petals/entity/exposures/openstreetmap/osm_dataloader.py

```diff
@@ -0,0 +1,945 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Define functions to download openstreetmap data
+"""
+import geopandas as gpd
+import logging
+from osgeo import ogr, gdal
+from pathlib import Path
+import shapely
+import subprocess
+from tqdm import tqdm
+import urllib.request
+import numpy as np
+import itertools
+import time
+import overpy
+
+from climada import CONFIG
+
+LOGGER = logging.getLogger(__name__)
+gdal.SetConfigOption("OSM_CONFIG_FILE",
+                     str(Path(__file__).resolve().parent.joinpath('osmconf.ini'))) 
```

Comment:
i'd suggest specifying this path in a config file rather than constructing it within the code. it's less likely to go unexpectedly wrong on weird operating systems and lets people write their own osmconf.ini

Reply:
good point. changed this now. 8b95b40d